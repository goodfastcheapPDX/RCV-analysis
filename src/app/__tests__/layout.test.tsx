import { render } from "@testing-library/react";
import { describe, expect, it, vi } from "vitest";
import RootLayout from "../layout";

// Mock AppShell component
vi.mock("@/components/AppShell", () => ({
  AppShell: ({ children }: { children: React.ReactNode }) => (
    <div data-testid="app-shell">{children}</div>
  ),
}));

describe("Root Layout", () => {
  it("should match snapshot", () => {
    const { container } = render(
      <RootLayout>
        <div>Test content</div>
      </RootLayout>,
    );
    expect(container.firstChild).toMatchSnapshot();
  });
});
