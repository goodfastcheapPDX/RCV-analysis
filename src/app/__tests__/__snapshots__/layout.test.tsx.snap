// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Root Layout > should match snapshot 1`] = `
<div
  data-testid="app-shell"
>
  <div>
    Test content
  </div>
</div>
`;
